iree/compiler/Dialect/VM/Target/C/CModuleTarget.cpp

```diff
@@ -15,6 +15,7 @@
 #include "iree/compiler/Dialect/VM/Target/C/CModuleTarget.h"
 
 #include "emitc/Target/Cpp.h"
+#include "iree/base/api.h"
```

Comment:
we can't have the compiler depend on runtime files - iree_string_view_compare is just strncmp, which you can use here directly

Reply:
std::string::compare would work too, or am I missing something?

Reply:
should